apps/plasmo/src/components/Calendar/CalendarDay.tsx

```
@@ -0,0 +1,58 @@
+import React from "react"
```

Biome

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)